src/gedi_subset/subset.py

```diff
@@ -169,31 +171,24 @@ def main(
         resolve_path=True,
     ),
     doi=typer.Option(
-        "10.3334/ORNLDAAC/2056",  # GEDI L4A DOI, v2.1
-        help="Digital Object Identifier of collection to subset (https://www.doi.org/)",
+        ...,
+        help=(
+            "Digital Object Identifier (DOI) of collection to subset"
+            " (https://www.doi.org/)"
+            " Can be a specific DOI"
+            f" or one of these logical names: {', '.join(logicals.keys())}"
+        ),
     ),
     cmr_host: CMRHost = typer.Option(
         CMRHost.maap,
         help="CMR hostname",
     ),
     columns: str = typer.Option(
-        ",".join(
-            [
-                "agbd",
-                "agbd_se",
-                "l2_quality_flag",
-                "l4_quality_flag",
-                "sensitivity",
-                "sensitivity_a2",
-            ]
-        ),
+        None,
         help="Comma-separated list of columns to select",
     ),
     query: str = typer.Option(
-        "l2_quality_flag == 1"
-        " and l4_quality_flag == 1"
-        " and sensitivity > 0.95"
-        " and sensitivity_a2 > 0.95",
+        None,
```

Comment:
This should be `...` (not `None`) to make it a required field.

```suggestion
        ...,
```